cli/cmd/get.go

```diff
@@ -739,7 +736,10 @@ func resourceStatusesStr(resourcesRes *schema.GetResourcesResponse) string {
 	for i, title := range titles {
 		paddingWidth := maxTitleLen - len(title) + 3
 		padding := strings.Repeat(" ", paddingWidth)
-		out += title + ":" + padding + values[i] + "\n"
+		out += title + ":" + padding + values[i]
```

Comment:
Is there still a new line separating these lines? Was there double new lines before?

Reply:
there was a trailing newline on the last line